bookkeeper-server/src/main/java/org/apache/bookkeeper/util/DiskChecker.java

```diff
@@ -191,9 +215,8 @@ public float checkDir(File dir) throws DiskErrorException,
      *
      * @param diskSpaceThreshold
      */
-    @VisibleForTesting
-    void setDiskSpaceThreshold(float diskSpaceThreshold, float diskUsageWarnThreshold) {
-        validateThreshold(diskSpaceThreshold, diskSpaceThreshold);
+    public void setDiskSpaceThreshold(float diskSpaceThreshold, float diskUsageWarnThreshold) {
```

Comment:
I don't see this called anywhere in the patch.  Is there a reason to make it public?